qiskit/providers/ibmq/runtime/__init__.py

```diff
@@ -102,7 +103,7 @@
     qc.measure_all()
 
     # Execute the circuit using the "circuit-runner" program.
-    program_inputs = {'circuits': circuit, 'measurement_error_mitigation': True}
+    program_inputs = Info(circuits=circuit, measurement_error_mitigation=True)
```

Comment:
I think the doc can use some description on this new interface, specially about how one can use auto complete to see the program parameters.

Reply:
Updated, but should still revisit these example usages.